sunkit_pyvista/plotter.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, coordinate_frame=None):
         self._coordinate_frame = coordinate_frame
         self._plotter = pv.Plotter()
         self.camera = self._plotter.camera
+        self.all_meshes = {}
+        self.mesh_block = pv.MultiBlock()
```

Comment:
Whats the endgoal of storing them twice?

Reply:
`multiblock`  allows us to combine all the plots into a single block whereas `all_meshes` just holds a reference to the meshes in a dictionary

Reply:
I was wondering what this would do to the memory usage to store the meshes twice, over just using the dict we have stored to pass that back into a multiblock when we needed it?

Reply:
Yeah, the latter approach seems better but we would need a separate method to flatten all `values` of the dictionary to a single list before being passed to `MultiBlock`, shall I go ahead with that?

Reply:
Sure.